.github/workflows/test.yml

```diff
@@ -8,11 +6,21 @@ on:
     branches:
       - '**'
 
+name: Go
 jobs:
-  build:
-    runs-on: ubuntu-latest
+  test:
+    strategy:
+      fail-fast: false
```

Comment:
Any reason to change the default behavior from `true` to `false`?

Reply:
From experience with cross-platform repos, this is just a more sensible default in my opinion. Because it gives all builds the chance to fail, which surfaces more errors that one in a fail-fast scenario. And where Windows is involved, all the builds take much longer in any case... and that's where most of my errors are! So, no real reason, other than this basically being my default elsewhere.

Reply:
Since we'll not be paying for the execution time, I don't see any downsides on using fail-fast false. @xetorthio thoughts?